org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserCodeCompiler.java

```diff
@@ -44,6 +50,10 @@ public void compileProgram() throws InternalServerError, UserInitiatedException
 
     // diagnostics will include any compiler errors
     for (Diagnostic<? extends JavaFileObject> diagnostic : diagnostics.getDiagnostics()) {
+      if (diagnostic.getCode().startsWith(DIAGNOSTIC_CODE_DEPRECATED_WARNING_PREFIX)) {
```

Comment:
nit: can you add a comment as to why we skip these?

Reply:
Can do!